WORKSPACE

```diff
@@ -154,3 +152,16 @@ bind(
   name = "default-play-routes-compiler-cli",
   actual = "//default-compiler-clis:scala_2_12_play_2_7"
 )
+
+rules_pkg_version = "0.7.0"
```

Comment:
Is this specifically being added because of the capabilities freeze of @bazel_tools//tools/build_defs/pkg in 4+?
If so, is this because bazel 6 finally has enough drift to be incompatible with the bazel_tools version?

Reply:
Great question. I guess this change in particular might not have been necessary because it wasn't required for the updates to the rules_scala project.

Reply:
Can't hurt to do it still